src/Form/Control.php

```diff
@@ -70,7 +70,7 @@ class Control extends View
      *
      * @var bool
      */
-    public $readonly = false;
+    public $readOnly = false;
```

Comment:
This property is definitely only `bool` only, please strong type it, also `disabled`.